packages/flutter_client_sdk/lib/src/flutter_state_detector.dart

```diff
@@ -23,7 +23,8 @@ final class FlutterStateDetector implements StateDetector {
   Stream<NetworkState> get networkState => _networkStateController.stream;
 
   late final LDAppLifecycleListener _lifecycleListener;
-  late final StreamSubscription<ConnectivityResult> _connectivitySubscription;
```

Comment:
I am a little concerned about also requiring an upgrade to a package that has only been out a few days. 

Ideally we could support both for a bit. Probably not elegant, but I wonder if we could wrap these couple contact points. Accept a dynamic, if the item is not a list, then turn it into a list of one, if it is a list then forward it along.

Reply:
Definitely not elegant, but it's not a bad idea! I just updated this PR, it should do the trick.

I thought about writing a wrapper as you suggested, but it looks like it's really just this one little spot in the code that needs to handle connectivity_plus v5, so I thought I'd keep it simple.

Reply:
@maxlapides Looks great. Thank you for the conrtibution!